spec/factories/facts.rb

```diff
@@ -0,0 +1,9 @@
+require 'faker'
+
+FactoryBot.define do
+  factory :fact do
+    fact_text { Faker::ChuckNorris.fact }
+    likes { Faker::Number.number(digits: 3).to_i }
```

Comment:
Faker is such a great and helpful tool! Some fun extra info is that there is also a built in Ruby class called Random. It has some interesting functionality that can be fun to play around with!